### DISCUSSION TITLE
docker启动报错

### DISCUSSION THREAD

**Discussion by joyoner** [1715173329/UnblockNeteaseMusic#738] [Q&A] [answered]
启动参数：-e https://music.163.com -s -o qq kugou kuwo migu ytdlp bilibili 

在部分服务器上无法启动，docker logs 报如下错误：
Error: getaddrinfo EAI_AGAIN music.163.com
    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:79:26) {
  errno: -3001,
  code: 'EAI_AGAIN',
  syscall: 'getaddrinfo',
  hostname: 'music.163.com'
}

**Comment by pan93412** [comment, marked as answer]
https://stackoverflow.com/questions/67957292/docker-compose-throws-error-with-getaddrinfo-eai-again-registry-npmjs-org 可以參考一下
